src/hardware-mapping.cc

```diff
@@ -97,6 +97,21 @@ bool HardwareMapping::AddMotorMapping(LogicAxis axis, int motor,
   return true;
 }
 
+int  HardwareMapping::GetMotorFromAxis(int axis) {
+    int i = 0;
+    // Map back the steps to the correct logical axis.
+    uint8_t temp;
+    // Get the axis to driver bitmap
+    temp = axis_to_driver_[axis];
+    // while the first bit is not 1, iterate.
+    while ( ((temp & 0x01) != 0x01) && i < NUM_MOTORS) {
```

Comment:
Maybe rename i as m for 'motor'.

There is an extra space after the opening parenthesis of after while, but not space at the end of the expression. In general: don't add unnecessary space after the opening parenthesis.